3.Tests/UI/Hooks/Hooks.cs

```diff
@@ -39,11 +40,50 @@ public Hooks(ScenarioContext scenarioContext)
     }
 
     [BeforeTestRun]
-    public static void BeforeTestRun()
+    public static void BeforeTest()
     {
+        string logsDirectory = Path.Combine(TestContext.CurrentContext.TestDirectory, "Logs");
+        string latestFilePath = Path.Combine(logsDirectory, $"Latest{DateTime.Now:yyyyMMddHH}.txt");
+
+        if (File.Exists(latestFilePath))
```

Comment:
Not sure this if statement will ever be true. If you are generating a timestamp right on this hook, every time you run the tests you will get a new latestFilePath, so that file will never exist at this point.

Reply:
You're right, it will be true only if you run the tests in the same hour. I changed it to use a Regex that delete the files that contains "Latest" in the specified path. In that way it will always delete the file.